packages/api-fetch/src/index.js

```diff
@@ -68,16 +76,24 @@ function apiFetch( options ) {
 					throw invalidJsonError;
 				}
 
+				const responseClone = response.clone();
```

Comment:
Unit test fails with the following message:
> [TypeError: r[1] esponse.clone is not a function]

I bet you need to mock it because it is a part of the response.

Reply:
👍🏻